scripts/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# This script builds wheels for the API, SDK, and extension packages in the
+# dist/ dir, to be uploaded to PyPI.
+
+set -ev
+
+# Get the latest versions of packaging tools
+python3 -m pip install --upgrade pip setuptools wheel
+
+BASEDIR=$(dirname $(readlink -f $(dirname $0)))
+
+(
+  cd $BASEDIR
+  mkdir -p dist
+  rm -rf dist/*
+
+ for d in opentelemetry-api/ opentelemetry-sdk/ ext/*/ ; do
+   (
+     cd "$d"
+     python3 setup.py --verbose bdist_wheel --dist-dir "$BASEDIR/dist/"
```

Comment:
Shouldn't we use  [`pip wheel`](https://pip.pypa.io/en/stable/reference/pip_wheel/) instead of invoking `setup.py` directly?

Reply:
I couldn't find any guidance on whether `pip wheel`  or `setup.py bdist_wheel` is preferred. I know that `pip install` is preferred over `setup.py install` but I guess the arguments there don't apply to `pip wheel`. So I think it's fine either way.

Reply:
Looking at https://github.com/pypa/pip/blob/master/src/pip/_internal/commands/wheel.py, I think it's safe to say that pip wheel *just* calls setup.py is a gross oversimplification. But if we need all the extra work that pip does is a whole other question.

What would be useful is generating a source distribution first using setup.py and then generate a wheel from that using `pip wheel`. That would verify that our MANIFEST.in, etc. is working correctly.

Reply:
I added #252 for building a source distribution first. I'll have to dig into the docs to develop an opinion here.

Reply:
One complication to note here is that `pip wheel` isolates builds from the env.

`setup.py bdist_wheel` builds as expected:
```sh
$ (cd opentelemetry-sdk/; python3 setup.py bdist_wheel --dist-dir='../dist' sdist --dist-dir='../dist'); ls dist
```
```
...
opentelemetry-sdk-0.4.dev0.tar.gz  opentelemetry_sdk-0.4.dev0-py3-none-any.whl
```

`pip wheel` fails because it tries to pip-install dependencies instead of using the current env:
```sh
(cd opentelemetry-sdk/; python3 setup.py sdist --dist-dir='../dist'); (cd dist; pip wheel opentelemetry-sdk-0.4.dev0.tar.gz); ls dist
```
```
...
Processing ./opentelemetry-sdk-0.4.dev0.tar.gz
ERROR: Could not find a version that satisfies the requirement opentelemetry-api==0.4.dev0 (from opentelemetry-sdk==0.4.dev0) (from versions: 0.1a0, 0.2a0, 0.3a0)
ERROR: No matching distribution found for opentelemetry-api==0.4.dev0 (from opentelemetry-sdk==0.4.dev0)
opentelemetry-sdk-0.4.dev0.tar.gz
```